modules/sdk/src/main/scala/org/tessellation/sdk/cli/CliMethod.scala

```diff
@@ -86,6 +87,10 @@ trait CliMethod {
       ensureCheckInterval = 10.seconds
     )
   )
+  val snapshotSizeConfig: SnapshotSizeConfig = SnapshotSizeConfig(
+    maxSignaturesSizeInBytes = 102_400L,
```

Comment:
Why can't we derive the maximum size of a single signature and then calculate how much they will take based on the number of facilitators after facility step ends? After it ends there can't be any more facilitators added. Hardcoding to 100kb or 300 signers seems like wasting possibly available space.

Reply:
IMO forbidding to sign the snapshot by additional peer is no-go and this is what you and Krystian proposes 🤔

Reply:
The maximum amount of signatures is known during consensus and no more will be accepted anyway, so leaving it open to accept more signatures given that it will never be utilized is pointless. The count of facilitators calculated at the end of `Facility` step is effectively the max count of signatures for that consensus round and in consequence a snapshot.

Reply:
ok, I got your point and I agree with you. since we know the maximum possible number of peers that would sign the proposal, we can limit the size of signatures to this and not to the maximum possible number taken from seedlist/allowance list.

Reply:
I agree. I checked the size of single `SignatureProof` and I multiply it by facilitators number from `Facility` step. So the max cap depends on number of facilitators. We can of course have less because some facilitators may fail at next steps but at least we won't waste size in a hardcoded way.